scipy/special/orthogonal.py

```diff
@@ -157,18 +155,12 @@ def __call__(self, v):
     def _scale(self, p):
         if p == 1.0:
             return
-        try:
-            self._coeffs
-        except AttributeError:
-            self.__dict__['coeffs'] *= p
-        else:
-            # the coeffs attr is be made private in future versions of NumPy
```

Comment:
This comment seems weird given that coeffs attr was already private at the time this was written.